### PR TITLE
Fix drawer menu on mobile

### PR DESCRIPTION
**Pull request by siggakogk**
# Drawer menu mobile bug

[efnisyfirlit fer yfir fótinn og þú sérð ekki síðasta linkinn
](https://app.asana.com/0/0/1196335150068950/f)

## What

Drawer menu was hiding las link in footer on mobile

## Why

Bug fix

## Screenshots / Gifs

Before:
image

After:
image

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against master before asking for a review

### CODE REVIEW

File: libs/island-ui/core/src/lib/Footer/Footer.tsx [hunk 1]
```diff
@@ -31,6 +31,7 @@ interface FooterProps {
   hideLanguageSwith?: boolean
   showMiddleLinks?: boolean
   showTagLinks?: boolean
+  showDrawerMenu?: boolean
```
Comment (barabrian) [inline libs/island-ui/core/src/lib/Footer/Footer.tsx:34]: would this be more descriptive if it was something like "hasDrawerMenu" ? I get the feeling like this is showing or hiding a drawer menu

File: apps/web/layouts/main.tsx [hunk 2]
```diff
@@ -164,6 +165,7 @@ const Layout: NextComponentType<
             {wrapContent ? <Box width="full">{children}</Box> : children}
           </div>
         </MenuTabsContext.Provider>
+        {String(showDrawerMenu)}
```
Comment (barabrian) [inline apps/web/layouts/main.tsx:168]: I'm not sure what this is doing, are we casting boolean to string and showing it on the page?

**Review by barabrian — COMMENTED** [review]
Looks nice! I only have a few questions

File: libs/island-ui/core/src/lib/Footer/Footer.tsx (hunk 1 above)
Comment (siggakogk) [reply libs/island-ui/core/src/lib/Footer/Footer.tsx:34]: yes! makes sense

File: apps/web/layouts/main.tsx (hunk 2 above)
Comment (siggakogk) [reply apps/web/layouts/main.tsx:168]: ahhhh no good catch this was me debugging 🤦

**Review by barabrian — APPROVED** [review]
LGTM!

**Pull request merged**